src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/services/PrisonId.kt

```diff
@@ -0,0 +1,13 @@
+package uk.gov.justice.digital.hmpps.prisonersearch.services
+
+import io.swagger.v3.oas.annotations.media.Schema
+
+@Schema(description = "Search Criteria for all prisoners at a prison")
+
+data class PrisonId(
+  @Schema(description = "Prison Id, Prison Id or OUT or TRN", example = "MDI")
+  val prisonId: String?) {
+  @Schema(hidden = true)
+  fun isValid() = !(prisonId.isNullOrBlank())
```

Comment:
since this is just a simple validation - could you add a `@NotBlank` constraint to the class and make `prisonId` therefore not nullable?  You might also need to add `@Valid` to the parameter too...